### DISCUSSION TITLE
How does homebrew customize the installation instructions in GitHub Packages to be `brew install <formula>`?

### DISCUSSION THREAD

**Discussion by kevinlul** [Homebrew/discussions#5510] [Tap maintenance and Homebrew development]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Normally for the container registry, GitHub would suggest the command `docker pull ghcr.io/IMAGE_NAME`. However, for bottles, it actually shows `brew install FORMULA`, though switching to the OS / Arch tab shows the docker pull commands again.

e.g. https://github.com/Homebrew/homebrew-core/pkgs/container/core%2Fsqlite/221454310?tag=3.46.0

![image](https://github.com/user-attachments/assets/8166d122-8768-4129-a890-749350a19d7b)

How is this accomplished?

**Comment by SMillerDev** [comment]
I suspect it's https://github.com/Homebrew/brew/blob/62ebd31942d4fae7fc2b14f765a2e9c41a6b373b/Library/Homebrew/github_packages.rb#L305

**Reply by kevinlul** [reply]
So it's a custom arrangement with GitHub, setting label `com.github.package.type` to `homebrew_bottle`?

**Reply by SMillerDev** [reply]
I have no idea, but that sounds plausible. You'd really have to ask GitHub how their UI works.
